packages/cli/package.json

```diff
@@ -9,34 +9,39 @@
   },
   "license": "MIT",
   "type": "module",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "main": "./src/index.ts",
```

Comment:
The interesting thing with this setup (main pointing to `src/index.ts`, exports pointing to `dist/index.js`) is that TypeScript updates types immediately during development (because it looks at `main` and ignores `exports`), but bundlers look at `exports` and resolve to the build file, so we might not have to switch around main entries (between js/ts) when releasing the package but still have the same local devex